### PR TITLE
CE-3738 | Changed code to for non logged in user to see entry point

### PR DESCRIPTION
**Pull request by d34th4ck3r**
@Wikia/spitfires @perjg

### CODE REVIEW

File: extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ public static function onArticleSaveComplete(
 	public static function onGetRailModuleList( Array &$railModuleList ) {
 		global $wgTitle, $wgUser;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php]: `$wgUser` is not used anymore

**Comment by lukaszkonieczny** [comment]
LGTM

**Comment by kamilkoterba** [comment]
LGTM

**Comment by kamilkoterba** [comment]
Tested, LGTM

**Pull request merged**